nipype/interfaces/base.py

```diff
@@ -764,6 +764,8 @@ def __init__(self, **inputs):
             raise Exception('No input_spec in class: %s' %
                             self.__class__.__name__)
         self.inputs = self.input_spec(**inputs)
+        self.estimated_memory = 1
```

Comment:
I don't have one. We can keep GB (that's what you use now yes?). It would
be just nice to make it explicit in the name.
On Feb 24, 2016 7:30 AM, "Daniel Clark" <notifications@github.com> wrote:

> In nipype/interfaces/base.py
> <https://github.com/nipy/nipype/pull/1372#discussion_r53954787>:
>
> > @@ -764,6 +764,8 @@ def __init__(self, **inputs):
> >              raise Exception('No input_spec in class: %s' %
> >                              self.__class__.__name__)
> >          self.inputs = self.input_spec(**inputs)
> > +        self.estimated_memory = 1
>
> Sure. Is there a preference between gb or mb for units?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nipy/nipype/pull/1372/files#r53954787>.
>